### PR TITLE
Update WLH to use ListCollectorDrivingQuestion

### PR DESCRIPTION
**Pull request by iwootten**
Due to the complexity of the scenarios for the household composition, we can't have a driving question when there are multiple list collectors for the same list. 

To fully resolve links on the summary for the household list, the current household schema should be modified to use variants over a driving question.

Additionally, we should update the visitors list to use a driving question single list collector.

### What is the context of this PR?

https://github.com/ONSdigital/eq-survey-runner/pull/2348

### How to review 

Go through WLH within census for eng/nir. You should see different variants whether you have entered anyone to the household or not. Answering either no/yes to each additional person within each household/visitor list collector.

There shouldn't be any broken links on the summary.

### CODE REVIEW

**Comment by iwootten** [comment]
[Add person to household on section summary doesn't work when "No, no one usually lives here" is selected](https://trello.com/c/o0W9MfTh/3304-add-person-to-household-on-section-summary-doesnt-work-when-no-no-one-usually-lives-here-is-selected)

**Review by JamesGardiner — APPROVED** [review]

File: data-source/jsonnet/england-wales/household/lib/rules.libsonnet [hunk 1]
```diff
@@ -1,5 +1,17 @@
 local common_rules = import '../../lib/common_rules.libsonnet';
 
+local listIsEmpty(listName) = {
+  list: listName,
+  condition: 'equals',
+  value: 0,
+};
```
Comment (ajmaddaford) [inline data-source/jsonnet/england-wales/household/lib/rules.libsonnet:7]: This condition is used in `central_heating.jsonnet` and `renewables.jsonnet`, we should replace  them with `listIsEmpty`.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Review by ajmaddaford — CHANGES_REQUESTED** [review]
The visitors fix hasn't been applied to the NIR survey.

**Review by ajmaddaford — APPROVED** [review]
👍

**Pull request merged**